docs/devGuide/development/migratingToTypeScript.md

```diff
@@ -22,7 +22,7 @@ We have decided on structuring the commits as two commits: a "Rename" commit and
 
 ### Step 1: Preparation
 
-1. **If running, stop automatic compilation on change/save**. More on this at [Editing backend features](workflow.md#editing-backend-features) section in the Workflow page.
+1. **If running, stop automatic compilation on change/save**. More on this at [Editing backend features](workflow.md#editing-backend-features) section in the Workflow page. Note that any git hooks installed will also compile and build the backend - if git hooks have already been installed for your MarkBind development setup, you will need to bypass them during the "Rename" commit later.
```

Comment:
Is there anything that can/need be done at this step for the git hook? If not let's remove this since the part below mention the same thing.

Reply:
Makes sense, will do :+1:

Reply:
Removed in [dba39d1](https://github.com/MarkBind/markbind/pull/2151/commits/dba39d1ce84ce271a31f6c753110d6e966e92108)!